pkg/alertmanager/distributor_test.go

```diff
@@ -34,7 +34,7 @@ import (
 	"google.golang.org/grpc/health/grpc_health_v1"
 
 	"github.com/grafana/mimir/pkg/alertmanager/alertmanagerpb"
-	util_log "github.com/grafana/mimir/pkg/util/log"
+	mimir_test "github.com/grafana/mimir/pkg/util/test"
```

Comment:
Package names and aliases shouldn't have underscores, and IMO it would be more discoverable if it was a concatenation of last two packages, so I'd suggest going for `utiltest` as the alias here.